scripts/lib/libvirt/compute-config

```diff
@@ -32,6 +32,8 @@ def main():
                         help="Virtual Disk Directory (e.g /dev/cloud)")
     parser.add_argument("bootorder", type=str,
                         help="Boot Order (e.g. 2)")
+    parser.add_argument("numcontrollers", type=str, default="1",
```

Comment:
as I mentioned to @aplanas this looks like a good idea, but then the tests for this function expect a string.